forms-flow-theme/scss/_variables.scss

```diff
@@ -202,6 +202,9 @@ hr {
   margin-top: 0;
   height: 100vh !important;
 }
+.margin-0{
+  margin: 0 !important;
+}
```

Comment:
can we remove this and instead use 'm-0' bootstrap class to html where margin-0 is used @fahad-aot

Reply:
Updated